crypto/rsa/rsa_pk1.c

```diff
@@ -253,3 +256,120 @@ int RSA_padding_check_PKCS1_type_2(unsigned char *to, int tlen,
 
     return constant_time_select_int(good, mlen, -1);
 }
+
+int rsa_padding_check_PKCS1_type_2_TLS(unsigned char *to, size_t tlen,
+                                       const unsigned char *from, size_t flen,
+                                       int client_version, int alt_version)
```

Comment:
Please document the function.

Reply:
I added a comment at the front describing it.